forge-game/src/main/java/forge/game/card/Card.java

```diff
@@ -5296,14 +5296,14 @@ public void setDamageHistory(CardDamageHistory history) {
 
     public final int getTotalDamageReceivedThisTurn() {
         int sum = 0;
-        for (Pair<Card, Integer> p : game.getDamageDoneThisTurn(null, false, null, "Card.StrictlySelf", this, getController(), null)) {
-            sum += p.getRight();
+        for (Integer i : game.getDamageDoneThisTurn(null, false, null, "Card.StrictlySelf", this, getController(), null)) {
```

Comment:
should this be better stored in Card getDamageHistory too?

So instead of just one Table for the Damage Sources, maybe a second for the damage Targets?